openexp/_canvas/psycho.py

```diff
@@ -45,6 +45,18 @@
 	raise osexception(
 		u'PsychoPy is missing the GratingStim() class. Please update your version of PsychoPy! For installation instructions, please visit http://www.psychopy.org/.')
 
+import psychopy, warnings
+from distutils.version import StrictVersion
+try:
+    PsyPy_vers = StrictVersion(psychopy.__version__)
+except ValueError:
+    warnings.warn(u'Invalid version number %s of PsychoPy'%psychopy.__version__)
```

Comment:
It's not visible here, but you're mixing space- and tab-based indentation. That's a *big* no! Only tab-based indentation in OpenSesame. The recommended Python-style (PEP8) is space-based, so you can use that for new projects too. But, in any case, never mix!